main.go

```diff
@@ -180,12 +180,21 @@ func handleRequest(resp http.ResponseWriter, req *http.Request, data_dir, map_na
 	}
 }
 
+func getSocketType(renderd_socket string) (string, *net.TCPAddr) {
+	tcp_addr, _ := net.ResolveTCPAddr("tcp", renderd_socket)
+	if tcp_addr != nil {
+		return "tcp", tcp_addr
+	} else {
+		return "unix", tcp_addr
```

Comment:
Might be easier to read
```suggestion
		return "unix", nil
```

Reply:
Yeah, I thought about that myself as it does indeed make things a bit clearer. Thanks!